source/ddbus/thin.d

```diff
@@ -27,6 +26,19 @@ struct ObjectPath {
     return _value;
   }
 
+  /// Returns the string representation of this ObjectPath
+  string value() const pure nothrow @safe {
+    return _value;
+  }
+
+  /// Sets and validates the string representation of this ObjectPath
+  string value(string newVal) @safe {
+    enforce(_isValid(newVal));
+    return _value = newVal;
+  }
+
+  alias value this;
```

Comment:
See previous comment

Reply:
However other languages implement this as string alternative (it is handled the same way as a string in dbus after all) so I think this `alias this` should stay